testsuite/billing.py

```diff
@@ -0,0 +1,53 @@
+"""APIs for Stripe and Braintree"""
+import braintree
+import stripe
+
+
+# pylint: disable=too-few-public-methods
+class Stripe:
+    """API for Stripe"""
+
+    def __init__(self, api_key):
+        stripe.api_key = api_key
```

Comment:
??? :-O This means all the stripe tests are disruptive?

Reply:
onky one stripe per 3scale only one api key for all tests, despite to fact it is disruptive, actually it doesn't have to be probably as it will be only one and still same api key

Reply:
coment is enough